cdap-app-templates/cdap-etl/cdap-etl-tools/pom.xml

```diff
@@ -71,7 +71,9 @@
     </resources>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
+        <version>2.4.3</version>
```

Comment:
If I leave this change in, `maven` complains:
```
[WARNING] Some problems were encountered while building the effective model for co.cask.cdap:cdap-etl-tools:jar:3.5.5
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-shade-plugin is missing. @ co.cask.cdap:cdap-etl-tools:[unknown-version], /Users/john/Source/cdap_3.5/cdap-app-templates/cdap-etl/cdap-etl-tools/pom.xml, line 73, column 15
```
and then the build eventually hangs.